data/mods/ssb/conditions.ts

```diff
@@ -2143,7 +2143,7 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
 		name: "Big Storm Coming Mod",
 		duration: 1,
 		onBasePower() {
-			return this.chainModify([0x4CC, 0x1000]);
+			return this.chainModify([1228, 4096]);
```

Comment:
You PMed me this earlier, and yeah I think it should be 1229 over 1228. `0.300...` is better than `0.299...`
```suggestion
			return this.chainModify([1229, 4096]);
```
for your convenience